tests/test_evaluator.py

```diff
@@ -205,6 +240,7 @@ def test_pipe_init(self):
         )
         self.assertEqual(results["accuracy"], 1.0)
 
+    @slow
```

Comment:
when possible you can mock a model using a rule based function

Reply:
That's what we do for all the other tests, but I think we can keep a slow test that loads an actual model. What do you think?